src/main/java/frc/robot/subsystems/AssistedBalance.java

```diff
@@ -0,0 +1,61 @@
+/*// Copyright (c) FIRST and other WPILib contributors// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+ 
+package frc.robot.subsystems;
```

Comment:
If we aren't using this subsystem, go ahead and remove it.

If we change our mind, we can use github to find it again

Reply:
Done